### PR TITLE
Avoid mocking Android Views in DynamicPropsTest

### PR DESCRIPTION
**Pull request by chggr**
## Summary
The testDynamicElevationApplied() test in DynamicPropsTest is currently
using Mockito to mock an Android View. This change removes mocking and
uses a fake LithoView extension instead to retrieve the set elevation.

## Changelog

Remove the need to mock an Android View in DynamicPropsTest

## Test Plan

This change only affects tests and is not user visible

### CODE REVIEW

File: litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java [hunk 1]
```diff
@@ -228,21 +221,40 @@ public Component build() {
           protected void setComponent(Component component) {}
         };
 
+    // We are using an old version of Robolectric which has a very limited
```
Comment (colriot) [inline litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java:224]: We've recently migrated all our tests to the 4.x version, so I wonder if this is still true at all. Can you check if a plain LithoView would be enough, please?

**Review by colriot — CHANGES_REQUESTED** [review]
@chggr one thing that I'd want to make sure before accepting this. We actually might be able to remove this workaround completely 🤔

File: litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java (hunk 1 above)
Comment (chggr) [reply litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java]: That is a great point! I have just used a plain LithoView and it worked beautifully. Thanks for the suggestion

Comment (colriot) [reply litho-it/src/test/java/com/facebook/litho/DynamicPropsTest.java]: Thank you for fixing this, Christos !

**Comment by colriot** [comment]
Merging

**Pull request closed without merging**